src/ensembl/src/content/app/custom-download/components/overlay/Overlay.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Several points:
- This component is in custom download, but its name is very generic. Perhaps change it to CustomDownloadContentOverlay? Unless it's a component that wants to be shared
- It looks like what the Overlay component is doing is achievable with the `filter: opacity(0.3)` CSS rule applied to  the `CustomDownloadContent` component. On the plus side, that makes the overlay component unnecessary here. On the minus side, filter isn't supported on IE11. But we don't much care for IE11 either.

Reply:
> It looks like what the Overlay component is doing is achievable with the filter: opacity(0.3)

It also prevents interaction with the elements behind the overlay.

Reply:
The naming issue for this component still stands. We can choose between:
- keeping the Overlay name (or change it to something as general, such as PopupBackground), and move the component to shared components, or
- keep the component in CustomDownload, but change its name (CustomDownloadOverlay? CustomDownloadPopupBackground?)

Reply:
I was thinking about moving it to shared components by adding `className` prop. Would that be okay?

Reply:
I think so.

Reply:
That's done